### DISCUSSION TITLE
How do I access files on an attached drive?

### DISCUSSION THREAD

**Discussion by crozilla** [IceWhaleTech/CasaOS#353] [Q&A]
CasaOS recognizes the drives, but I can't get to the files on them. What am I missing?

**Comment by Lauren-ED209** [comment]
would you please share more about your environment? like disk type, hardware connection, file system and so on..

**Reply by crozilla** [reply]
Sure, using a Raspberry Pi 4 w/8GB RAM inside an Argon40 EON case. RasPi 64Bit Lite, then ran Argon's scripts (to make LED work and config fans) before installing CasaOS. Most stuff works fine, just don't know how to access files on my attached Samsung SSD. The drive appears in the sidebar (but says "Used: N/A"), and when I click on the gear, the Storage Manager only shows the boot disc (my SanDisk micro 120GB). Thoughts?
